### PR TITLE
Basic layout for Experiments page

### LINKED ISSUE #11: Implement Experiments page
## Description
* Implement experiments page

## Design
https://github.com/AlexsLemonade/refinebio/issues/246
https://projects.invisionapp.com/d/main#/console/13557531/297243710/preview

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

close #11 
https://github.com/AlexsLemonade/refinebio/issues/246

## Purpose/Implementation Notes

Added basic layout for the Experiments Page. We broke the missing parts into: #21 #22 

## Types of changes

* [ ] Bugfix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots

image

### CODE REVIEW

File: src/containers/Experiment/index.js [hunk 1]
```diff
@@ -1,17 +1,172 @@
...
+                </div>
+                <div>{experiment.submitter_institution}</div>
+              </div>
+              <div className="experiment__row">
+                <div className="experiment__row-label">Submitter</div>
+                <div>{experiment.pubmed_id}</div>
+              </div>
+            </div>
+
+            <hr className="experiment__divider" />
```
Comment (ramenhog) [inline src/containers/Experiment/index.js]: hmmm, might want to do this border in CSS: https://stackoverflow.com/questions/3513576/correct-usage-of-html5-hr-tag?noredirect=1&lq=1

File: src/components/Loader/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// </Loader>
+export default class Loader extends React.Component {
+  state = {
+    isLoading: true,
+    data: null
+  };
+
+  async componentDidMount() {
+    this.setState({ isLoading: true });
+    let data = await this.props.fetch();
```
Comment (ramenhog) [inline src/components/Loader/index.js]: `const`?

File: src/state/experiment/actions.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const loadExperiment = data => ({
+  type: 'LOAD_EXPERIMENT',
+  data
+});
+
+export const fetchExperiment = id => async dispatch => {
+  let data = await (await fetch(`/experiments/${id}.json`)).json();
```
[the remark this answers is not in the document]
Comment (ramenhog) [reply src/state/experiment/actions.js]: I think you need the trailing backslash: `/experiments/${id}/` not `/experiments/${id}`

[the remark this answers is not in the document]
Comment (arielsvn) [reply src/state/experiment/actions.js]: worked now, thanks! :)

File: src/components/Loader/index.js (hunk 2 above)
Comment (arielsvn) [reply src/components/Loader/index.js]: I realized I have to use `const` more often :)

**Comment by arielsvn** [comment]
@ramenhog I addressed the comments, let me know what do you think.

File: src/common/styles/_variables.scss [hunk 4]
```diff
@@ -11,6 +11,9 @@ $azure: #386DB0;
 // FUNCTIONAL COLORS
 $font-color: $black;
 $background-color--light: $whisper;
+$background-color--highlight: #F2F2F2;
```
[the remark this answers is not in the document]
Comment (ramenhog) [reply src/common/styles/_variables.scss]: ahh, I see, my mistake, we can keep it as highlight then 👍

**Comment by ramenhog** [comment]
The code looks good! Just one last fix: when I functionally tested it on my computer, I noticed that this happens when there are titles:

screen shot 2018-05-23 at 2 05 35 pm

So we probably just want a `max-width` on the title container.

**Comment by arielsvn** [comment]
Good catch @ramenhog, I just pushed a fix for the title. Let me know what do you think.

image

**Pull request merged**